app/vmalert/notifier/alertmanager.go

```diff
@@ -59,8 +61,22 @@ func (am AlertManager) Addr() string {
 
 // Send an alert or resolve message
 func (am *AlertManager) Send(ctx context.Context, alerts []Alert, headers map[string]string) error {
-	am.metrics.alertsSent.Add(len(alerts))
-	err := am.send(ctx, alerts, headers)
+	var sendAlerts []Alert
+	for _, alert := range alerts {
+		labels := alert.toPromLabels(am.relabelConfigs)
```

Comment:
it doesn't look like we send these relabeled labels after. Does it mean relabeling can't modify alert labels, only drop them completely?

Reply:
If label values can be modified and this is just a typo, would you mind adding relevant test case?

Reply:
It was a bug, fixed and added ut, plz check.